praw/models/reddit/subreddit.py

```diff
@@ -17,6 +17,117 @@ class Subreddit(RedditBase, MessageableMixin, SubredditListingMixin):
+            the default and trending lists.
+        :param collapse_deleted_comments: Collapse deleted and removed comments
+            on comments pages by default.
+        :param comment_score_hide_mins: The number of minutes to hide comment
+            scores.
+        :param description: Shown in the sidebar of your subreddit.
+        :param domain: Undocumented.
+        :param exclude_modqueue_banned: Exclude posts by site-wide banned users
+            from modqueue/unmoderated.
+        :param header_hover_text: The text seen when hovering over the snoo.
+        :param hide_ads: Don't show ads within this subreddit. Only applies to
+            gold-user only subreddits.
+        :param key_color: A 6-digit rgb hex color (e.g. `#AABBCC`), used as a
+            thematic color for your subreddit on mobile.
+        :param lang: A valid IETF language tag (underscore separated).
+        :param link_type: The types of submissions users can make.
+            One of ``any``, ``link``, ``self``.
+        :param name: The name for the new subreddit. Required during creation.
+        :param over_18: Viewers must be over 18 years old.
+        :param public_description: Public description blurb. Appears in search
+            results and on the landing page for private subreddits.
+        :param public_traffic: Make the traffic stats page public.
+        :param show_media: Expand media previews on comments pages.
+        :param spam_comments: Spam filter strength for comments.
+            One of ``low``, ``high``, ``all``.
+        :param spam_links: Spam filter strength for links.
+            One of ``low``, ``high``, ``all``.
+        :param spam_selfposts: Spam filter strength for selfposts.
+            One of ``low``, ``high``, ``all``.
+        :param sr: The fullname of the subreddit whose settings will be
+            updated.
+        :param submit_link_label: Custom label for submit link button
+            (blank for default).
+        :param submit_text: Text to show on submission page.
+        :param submit_text_label: Custom label for submit text post button
+            (blank for default).
+        :param subreddit_type: One of ``gold_restricted``, ``archived``,
+            ``restricted``, ``gold_only``, ``employees_only``, ``private``,
+            ``public``.
+        :param suggested_comment_sort: All comment threads will use this
+            sorting method by default. Leave blank, or choose one of
+            ``confidence``, ``top``, ``new``, ``controversial``, ``old``,
+            ``random``, ``qa``.
+        :param title: The title for the new subreddit.
+        :param wiki_edit_age: Account age (days) required to edit and create
+            wiki pages.
+        :param wiki_edit_karma: Subreddit karma required to edit and create
+            wiki pages.
+        :param wikimode: One of ``disabled``, ``modonly``, ``anyone``.
+
+        Any parameters left `None` will take on a default value assigned by
```

Comment:
Can we say, "Any keyword parameters not provided, or explicitly set to None, will ... assigned by the Reddit server"?